cou/utils/openstack.py

```diff
@@ -39,6 +39,8 @@
     "mysql": ["mysql-innodb-cluster", "mysql-router"],
 }
 
+DATA_PLANE_CHARMS = ["nova-compute", "ceph-osd", "swift-proxy", "swift-storage"]
```

Comment:
just a note: I don't remember how we upgrade swift-proxy, I suggest getting in touch with Gui. Swift-proxy is arguably a data plane app, but its unavailability does not cause loss of connectivity for VMs, which is usually what we are worried about

Reply:
After checking with Giuseppe I've removed "swift-proxy", "swift-storage" from data-plane.